docs/deployment.md

```diff
@@ -0,0 +1,56 @@
+# Deployment
+
+*This section is under construction*
```

Comment:
hm, it was not my intention to make that notice so "in your face" as admonition does.